src/Sulu/Bundle/MediaBundle/Admin/MediaAdmin.php

```diff
@@ -59,7 +59,7 @@ public function getJsBundleName()
     public function getRoutes(): array
     {
         return [
-            new Route('sulu_media.overview', '/media', 'sulu_media.overview'),
+            new Route('sulu_media.overview', '/media/collections/:id?', 'sulu_media.overview'),
```

Comment:
@chirimoya What URL should we choose here? I think to be consistent with what we already decided it should be `/collections/:id?`

Reply:
yes, go with `/collections/:id?`